### ISSUE TITLE
Feature : expose TrackError

### ISSUE THREAD

**Issue by calca** [Microsoft/appcenter-sdk-cordova#46]
### **Description**

Expose TrackError function in Crashes Cordova Plugins

**Comment by elamalani** [comment]
@calca Thanks for using App Center. Currently, we expose handled error functionality only for Xamarin apps. It's on our roadmap to support it for other platforms but there is no ETA yet. We will keep this thread open for visibility and keep you posted once we plan to support it for Cordova.

**Comment by calca** [comment]
@elamalani the API is not available on Android / iOS Native library ?

**Comment by enko** [comment]
This would be very good to have. Currently I'm abusing analytics, but that is rather limited, as the events are truncated and especially stacktraces go missing.

**Comment by elamalani** [comment]
@calca No, the API is not available for native iOS and Android. It's only available in Xamarin SDK right now.

**Comment by hoshauch** [comment]
Hi, we want to switch from IonicPro to AppCenter, so we need the Errorlogging plz :)

**Comment by conniey** [comment]
@elamalani 
What is the workaround for this? Currently my application has [ApplicationInsights-js](https://github.com/Microsoft/ApplicationInsights-JS) and is using its [trackException](https://github.com/Microsoft/ApplicationInsights-JS/blob/master/API-reference.md#trackexception) for exceptions. I've tried to remove that dependency and go to App Center SDK then export data to AI.

Is this the only way to track handled exceptions?

**Comment by elamalani** [comment]
@conniey You could use TrackEvent() API to track handled exceptions for now.

**Comment by conniey** [comment]
Thanks! When I export them to Application Insights, will I be able to easily correlate my client-side between any server-side issues? (I have an ASP.NET Core backend that writes telemetry to the same AI instance.)

**Comment by alexeystrakh** [comment]
hi @elamalani, do you have any updates on this feature request, as @enko confirmed, tracking errors with events is possible but we still lose stack trace information:

https://github.com/microsoft/appcenter-sdk-cordova/issues/46#issuecomment-403814895
